isb/Dockerfile

```diff
@@ -15,7 +15,7 @@ COPY ./isamples_webui/ ./
 ARG ISB_SITEMAP_PREFIX=https://mars.cyverse.org
 ARG ANALYTICS_DOMAIN=isamples.org
 # write out the JSON config for the specific solr URL to hit in the webUI
-RUN echo "{\n    \"solr_url\": \"$ISB_SITEMAP_PREFIX/thing/select\",    \"analytics_src\": \"https://metrics.isample.xyz/js/plausible.js\",\n    \"analytics_domain\": \"$ANALYTICS_DOMAIN\"\n}" | tee ./src/config.json
+RUN echo "const _solr_base = \"$ISB_SITEMAP_PREFIX/thing/\";\n\nconst config = {\n  \"solr_url\": _solr_base + \"select\",\n  \"solr_stream\": _solr_base + \"stream\",\n  \"analytics_src\": \"https://metrics.isample.xyz/js/plausible.js\",\n  \"analytics_domain\": \"isamples.org\"\n};\n\nexport default config;" | tee ./src/config.js
```

Comment:
The new string of `config.js`. Using export default config in the last to solve the bug in `npm run build --if-present`. @dannymandel I have a question. Should I add `isamples_central` between `ISB_SITEMAP_PREFIX` and `/thing`?

Reply:
No, `ISB_SITEMAP_PREFIX` would contain `isamples_central` as part of the string if we were building for an `isamples_central` configuration.